.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 script:
   - docker run
     -v $(pwd):/repo
-    -e PROJECT=hollowverse-perf-monitor
+    -e PROJECT=hollowverse-track-performance
```

Comment:
Is this change ok?

Reply:
This variable is no longer used. This whole line can be removed. It's from the Google App Engine days :sweat_smile: